### DISCUSSION TITLE
Release v1.2.8b / 5.57.7

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2104] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

### Fixed
- fixed issue with context menu setup on windows 11
- fixed column issue in vintage mode [#2103](https://github.com/sandboxie-plus/Sandboxie/issues/2103)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.2.9'>Release v1.2.8b / 5.57.7</a>.</em>

**Comment by BilleBarrett** [comment]
As soon as I brought up Chrome
My system crashed with KERNEL SECURITY CHECK FAILURE
Did it 3 times and then I went back to v1.28 and I'm OK
Correction V1.28 is now causing my system to crash.
I look at it tomorrow ow

**Reply by DavidXanatos** [reply]
well 1.2.8 and 1.2.8b have no changes to the driver only minor UI changes in sandman, so the crash must have an untelated cause
